upgrade/helpers/tasks.py

```diff
@@ -573,6 +577,7 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
                     host=host)
         else:
             host = entities.Host().search(query={'search': 'name={}'.format(host)})[0]
+            logger.info(f"Adding the Subscription {sub.name} on {host}")
```

Comment:
```suggestion
            logger.info(f"Adding the Subscription {sub.name} on {host=}")
```

Reply:
Faced some issue with Travis, so I rolldcback the changes